packages/legend-application-studio/src/stores/editor/utils/ModelClassifierUtils.ts

```diff
@@ -73,3 +73,13 @@ export enum PACKAGEABLE_ELEMENT_TYPE {
   TEMPORARY__LOCAL_CONNECTION = 'LOCAL_CONNECTION',
   INTERNAL__UNKNOWN = 'UNKNOWN',
 }
+
+export enum PACKAGEABLE_ELEMENT_GROUP_BY_CATEGORY {
+  MODEL = 'Model',
+  DATA_SPACE = 'Data Space',
+  EXTERNAL_FORMAT = 'External Format',
```

Comment:
I don't think we should leak information about having categories (`ExternalFormat`, `DataSpace`) from extensions here. We should bring such information via extensions too.

Reply:
For this external format is fine to be in core, but we should extend dataspace